src/iotjs_objectwrap.c

```diff
@@ -17,44 +17,30 @@
 #include "iotjs_objectwrap.h"
 
 
-namespace iotjs {
+void iotjs_jobjectwrap_initialize(iotjs_jobjectwrap_t* jobjectwrap,
+                                  const iotjs_jval_t* jobject,
+                                  uintptr_t jhandle,
```

Comment:
Good point! JerryScript already has a dependency of C99 since it uses `stdbool.h` so It would be natural that IoT.js also follow C99 dependency. 
I am considering to replace `typedef enum { false, true } bool;` with `<stdbool.h>` but not this time :)